4001-5000/LIT4155MiraclesMichael.xml

```diff
@@ -52,12 +52,56 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-12-16">Created XML record from Ethio authority google spreadsheet</change>
          <change who="DR" when="2017-04-02">Added keywords and adjustments</change>
          <change who="MV" when="2017-05-10">Updated keyword</change>
+         <change when="2024-12-12" who="ES">added text divs</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="bibliography">
+            <listRelation><relation name="ecrm:P129_is_about" active="LIT3959Homily" passive="PRS7049Michael"/></listRelation>
             
+            <!-- <note>???? Which miracle??? Variant of the <ref type="work" corresp="LIT3959Homily">homily</ref> by <persName ref="PRS3828Epiphani"/> on the angels and the child <persName ref="PRS11321Talafinos"/> transmitted also as <ref type="work" corresp="LIT1295Dersan#HomilyTalafinos"/>, 
+                  <ref type="work" target="LIT1295Dersan#CommemorativeTemple">Commemorative note for Sane</ref> and 
+                  <ref type="work" target="LIT1295Dersan#HomilyTalason">Homily for Hamle on Talason</ref>. </note> -->
+         </div>
+         <div type="edition">
+            <div type="textpart" xml:id="Miracle1" corresp="LIT1295Dersan#MiracleSailors">
```

Comment:
The order of miracles is probably often mixed or incomplete. Why do you not select the same xml:id as in LIT1295Dersan (e.g. MiracleSailors)

Reply:
When someone checks the miracles - sure. I now assume the standard order by month in the miracle collection (which is not Dersan) and only list the standard set. I however avoid the month names. Same IDs may be confusing, the link is not the same.

I have now created individual records and added corresp for all miracles here and in DM